M2/Macaulay2/packages/Permutations.m2

```diff
@@ -0,0 +1,98 @@
+newPackage(
+    "Permutations",
+    AuxiliaryFiles => true,
+    Version => "1.0", 
+    Date => "June 4, 2024",
+    Keywords => {"Combinatorics"},
+    Authors => {
+        {Name => "Sean Grate", 
+         Email => "sean.grate@auburn.edu", 
+         HomePage => "https://seangrate.com/"}
+  },
+    Headline => "functions for working with permutations",
+    PackageExports => {
+        -- "SimplicialComplexes",
+        -- "SimplicialDecomposability",
+        -- "Posets",
+        -- "MinimalPrimes"
```

Comment:
Is there a reason all these packages in `PackageExports` are commented out?

Reply:
No, a byproduct of copying from MatrixSchubert I believe. Should be fixed in some new commits.